### DISCUSSION TITLE
CFU-like data

### DISCUSSION THREAD

**Discussion by latlatina** [MIT-LCP/mimic-code#1567] [MIMIC-General] [answered]
Hi,

I hope it's OK to ask - I'm looking for a specific type of information, and I want to make sure it is available before I go through training and the rest of the procedure...
Does anyone know if there is information regarding CFU, or any type of bacterial load?
If I understood correctly, the microbiologyevents contains info about species, but not their amount, or abundance. Is it correct?

Thanks

**Comment by alistairewj** [comment]
Yes AFAIK that's correct. *microbiologyevents* contains: test name, isolate number, organism, antibiotic tested, and dilution level.

**Reply by latlatina** [reply]
Thank you,
If I understood correctly, the dilution level corresponds to the antibiotics and not the bacteria, right? It's not a value I can use to calculate bacterial abundance (CFU)...?

**Reply by alistairewj** [reply, marked as answer]
Yes correct, the dilution is for the antibiotic.

Double checking myself, as usual there is *some* information in the deidentified free-text comment associated with microbiology cultures:

```sql
select comments, count(*) as n
from `physionet-data.mimiciv_hosp.microbiologyevents`
where lower(comments) like '%cfu%'
group by 1 order by 2 desc
```

comments | n
--- | ---
"< 10,000 CFU/mL.  " | 24087
"10,000-100,000 CFU/mL Commensal Respiratory Flora.  " | 4019
"NO GROWTH, <1000 CFU/ml.  " | 2882
">100,000 CFU/mL Commensal Respiratory Flora.  " | 1188
~1000 CFU/mL Commensal Respiratory Flora.  | 475
~2000 CFU/mL Commensal Respiratory Flora. | 425
~3000 CFU/mL Commensal Respiratory Flora. | 244

But the *microbiologyevents* table has 3,228,713 rows so this is a very small proportion.

**Reply by latlatina** [reply]
Thank you for making sure. I really appreciate it.
